sys/core/src/TxsCore.hs

```diff
@@ -350,11 +354,14 @@ txsEval vexp  =  do
                              writeEnvBtoEnvC envb'
                              return wal'
 
+-- | Type for solve (@txsSolve, @txsUniSolve, and @txsRanSolve)                             
```

Comment:
Since Server needs the type `TxsDefs.VExpr  -> IOC.IOC (TxsDefs.WEnv VarId)`,
I consider the current solution the best of the many evils.
I didn't want to add a dependency on IOC in server.